modules/swagger-codegen-cli/src/main/java/com/wordnik/swagger/codegen/cmd/Generate.java

```diff
@@ -64,6 +70,11 @@ public void run() {
             input.setAuth(auth);
         }
 
+        if (isNotEmpty(basicAuth)) {
+            String authorization = new String(Base64Coder.encode(basicAuth.getBytes()));
+            input.setAuth("Authorization:" + "Basic "+ authorization);
```

Comment:
I think what @lanwen suggested is the following (which concatenates the strings once only):
`input.setAuth("Authorization: Basic "+ authorization);`